crates/bevy_reflect/bevy_reflect_derive/src/reflect_trait.rs

```diff
@@ -22,6 +22,10 @@ impl Parse for TraitInfo {
     }
 }
 
+/// A trait attribute macro that allows a reflected type to be downcast to a trait object.
+///
+/// This generates a struct that takes the form `Reflect<Trait Name>`. An instance of this struct can then be
```

Comment:
```suggestion
/// This generates a struct that takes the form `ReflectMyTrait`. An instance of this struct can then be
```

The angle brackets are wicked confusing in this context because it makes me think the name would be `Reflect<MyTrait>` instead, as a generic type parameter.

Reply:
Good point!